### ISSUE TITLE
Add link to live version of the explorer with an upload area

### ISSUE THREAD

**Issue by MarcSkovMadsen** [holoviz/hvplot#856]
To give users something useful right away it would be quite powerful to provide a link to a live version (powered by pyodide) of the Explorer composed with a FileUpload and a CodeArea to copy paste.

![image](https://user-images.githubusercontent.com/42288570/186674496-95fc4220-31dc-4017-8e55-97ca8349312b.png)

**Comment by jbednar** [comment]
Definitely! I think a live example would be really useful even without the upload or paste support, just so that people can get an interactive experience of working with these tools.
